harness/tf-moz-deepspeech/src/lib.rs

```diff
@@ -11,6 +11,7 @@ use std::str::FromStr;
 use tract_tensorflow::prelude::*;
 
 #[allow(dead_code)]
```

Comment:
You can probably remove that line, this function wouldn’t be dead if it was only compiled for tests.

Reply:
Thanks, done.